examples/existing_vpc/README.md

```diff
@@ -0,0 +1,3 @@
+# Existing VPC and subnets Example
+
+A very basic example to provision SLZ VPC with existing vpc.
```

Comment:
It's not really a basic example. It's an example of calling the module with an existing VPC and subnets